src/test/java/com/github/tomakehurst/wiremock/matching/RequestPatternTest.java

```diff
@@ -35,437 +54,498 @@
 
 public class RequestPatternTest {
 
-    @Test
```

Comment:
Would be better to keep formatting of the original tests even if it violates the standard. Reviiews with `?w=1`  is a solution, of course

Reply:
If I keep it same, then spotless check will fail.